### PR TITLE
Add a pipeline that will run Nightly MSAL against Prod Broker Automation

### PR DESCRIPTION
**Pull request by shahzaibj**
Add a pipeline that will run Nightly MSAL against Prod Broker Automation

### CODE REVIEW

File: azure-pipelines/ui-automation/msal-broker-ProdBroker-nightly.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+stages:
+  # msalautomationapp
+  - stage: 'msalautomationapp'
+    displayName: Build MSAL Automation APKs
+    jobs:
+      - template: ./templates/build-msal-automation-app.yml
+        parameters:
+          brokerFlavor: AutoBroker
+          msalFlavor: Local
```
Comment (fadidurah) [inline azure-pipelines/ui-automation/msal-broker-ProdBroker-nightly.yml:73]: IIRC we are creating a separate pipeline for Prod msal?

**Review by fadidurah — APPROVED** [review]
LGTM, left one comment

File: azure-pipelines/ui-automation/msal-broker-ProdBroker-nightly.yml (hunk 1 above)
Comment (shahzaibj) [reply azure-pipelines/ui-automation/msal-broker-ProdBroker-nightly.yml:73]: Yes

**Review by wainaina — APPROVED** [review]
LGTM

**Pull request merged**